<def f='qtbase-6.5.0/src/corelib/io/qlockfile.h' l='44' ll='49'/>
<use f='qtbase-6.5.0/src/corelib/io/qlockfile.h' l='50' c='_ZNK9QLockFile5errorEv'/>
<use f='qtbase-6.5.0/src/corelib/io/qlockfile_p.h' l='46' c='_ZN16QLockFilePrivate11tryLock_sysEv'/>
<use f='qtbase-6.5.0/src/corelib/io/qlockfile_p.h' l='64'/>
<use f='qtbase-6.5.0/src/corelib/io/qlockfile.cpp' l='449' c='_ZNK9QLockFile5errorEv'/>
<doc f='qtbase-6.5.0/src/corelib/io/qlockfile.cpp' l='84'>/*!
    \enum QLockFile::LockError

    This enum describes the result of the last call to lock() or tryLock().

    \value NoError The lock was acquired successfully.
    \value LockFailedError The lock could not be acquired because another process holds it.
    \value PermissionError The lock file could not be created, for lack of permissions
                           in the parent directory.
    \value UnknownError Another error happened, for instance a full partition
                        prevented writing out the lock file.
*/</doc>
<use f='qtbase-6.5.0/src/corelib/io/qlockfile_unix.cpp' l='118' c='_ZN16QLockFilePrivate11tryLock_sysEv'/>
